test/e2e/storage/persistent_volumes-local.go

```diff
@@ -208,91 +226,34 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			podSpec2 := makeLocalPod(config, testVol, readCmd)
 			f.TestContainerOutput("pod reads PV", podSpec2, 0, []string{testVol.hostDir})
 		})
-
-		It("should be able to mount volume in two pods one after other, write from pod1, and read from pod2", func() {
-			By("Creating pod1")
-			pod1, pod1Err := createLocalPod(config, testVol)
-			Expect(pod1Err).NotTo(HaveOccurred())
-
-			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod1))
-			pod1NodeName, pod1NodeNameErr := podNodeName(config, pod1)
-			Expect(pod1NodeNameErr).NotTo(HaveOccurred())
-			framework.Logf("Pod1 %q created on Node %q", pod1.Name, pod1NodeName)
-			Expect(pod1NodeName).To(Equal(node0.Name))
-
-			writeCmd, readCmd := createWriteAndReadCmds(volumeDir, testFile, testVol.hostDir /*writeTestFileContent*/)
-
-			By("Writing in pod1")
-			podRWCmdExec(pod1, writeCmd)
-
-			By("Deleting pod1")
-			framework.DeletePodOrFail(config.client, config.ns, pod1.Name)
-
-			By("Creating pod2")
-			pod2, pod2Err := createLocalPod(config, testVol)
-			Expect(pod2Err).NotTo(HaveOccurred())
-
-			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, pod2))
-			pod2NodeName, pod2NodeNameErr := podNodeName(config, pod2)
-			Expect(pod2NodeNameErr).NotTo(HaveOccurred())
-			framework.Logf("Pod2 %q created on Node %q", pod2.Name, pod2NodeName)
-			Expect(pod2NodeName).To(Equal(node0.Name))
-
-			By("Reading in pod2")
-			readOut := podRWCmdExec(pod2, readCmd)
-			Expect(readOut).To(ContainSubstring(testVol.hostDir)) /*aka writeTestFileContents*/
-
-			By("Deleting pod2")
-			framework.DeletePodOrFail(config.client, config.ns, pod2.Name)
-		})
 	})
 
-	Context("when two pods request one prebound PVC at the same time", func() {
-
-		var testVol *localTestVolume
-
-		BeforeEach(func() {
-			testVol = setupLocalVolumePVCPV(config, node0)
-		})
-
-		AfterEach(func() {
-			cleanupLocalVolume(config, testVol)
-		})
-
-		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2", func() {
-			By("Creating pod1 to write to the PV")
-			pod1, pod1Err := createLocalPod(config, testVol)
-			Expect(pod1Err).NotTo(HaveOccurred())
+	Context("Test various types of local volumes", func() {
```

Comment:
Maybe there should be two contexts.

One for "when two pods request one prebound PVC one after other" and another for "when two pods request one prebound PVC at the same time".

Then you only loop over the volume types and not the test cases.